underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -409,11 +409,6 @@ protected String getRootKey() {
 
   @Override
   protected InputStream openObject(String key, OpenOptions options) throws IOException {
-    try {
-      return new S3AInputStream(mBucketName, key, mClient, options.getOffset());
-    } catch (AmazonClientException e) {
-      LOG.error("Failed to open file: {}", key, e);
-      return null;
-    }
+    return new S3AInputStream(mBucketName, key, mClient, options.getOffset());
```

Comment:
AmazonClientException is a runtime exception, is it ok to throw that from this method?